package.json

```diff
@@ -16,6 +16,7 @@
     "@types/react-dom": "^18.0.1",
     "axios": "^0.26.1",
     "babel-plugin-macros": "^3.1.0",
+    "dateformat": "^5.0.3",
```

Comment:
hmm I think this library is for nodejs, we might not need this library tho? I was able to do it using `Intl.DateTimeFormat` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat
or https://devhints.io/wip/intl-datetime for easier to read documentation